examples/bower_components/dom-scheduler/bower_components/fast-list/js/page.js

```diff
@@ -9,7 +12,8 @@
     var listContainer = document.querySelector('section');
 
     var source = new BaconSource();
-    var list = new ScheduledList(listContainer, source, scheduler);
+    var list = new ScheduledList(listContainer, source);
+    var scheduler = ScheduledList.scheduler;
```

Comment:
when do we load this file as part of this repo ?
(none of the example app uses it I think)